### ISSUE TITLE
3.2.0: `setup.py clean` fails with `NameError: name 'long_description' is not defined`

### ISSUE THREAD

**Issue by sandrotosi** [matplotlib/matplotlib#16704]
Hello,
when preparing 3.2.0 for debian i noticed that running `setup.py clean` fails with

```
Traceback (most recent call last):
  File "./setup.py", line 247, in <module>
    long_description=long_description,
NameError: name 'long_description' is not defined
```

That's because https://github.com/matplotlib/matplotlib/blob/master/setup.py#L228 is only executed when https://github.com/matplotlib/matplotlib/blob/master/setup.py#L178-L180 and so it's not available when needed at https://github.com/matplotlib/matplotlib/blob/master/setup.py#L246

**Comment by sandrotosi** [comment]
de-indenting one level https://github.com/matplotlib/matplotlib/blob/master/setup.py#L226-L228 (so at the same level of `setup()`) fixes it

**Comment by QuLogic** [comment]
That looks correct; I've incorporated your fix into #16671.
